ci/Makefile

```diff
@@ -13,7 +13,7 @@ PYTHON := PYTHONPATH=$${PYTHONPATH:+$${PYTHONPATH}:}$(EXTRA_PYTHONPATH) python3
 check:
 	$(PYTHON) -m flake8 ci
 	$(PYTHON) -m pylint --rcfile ../pylintrc ci --score=n
-	$(PYTHON) -m black ci --check --line-length=120 --skip-string-normalization
+	$(PYTHON) -m black ci --check --diff --line-length=120 --skip-string-normalization
```

Comment:
what's this?

Reply:
I was getting errors in this check. `--diff` actually gives you the difference on what your original files looked like and what the rewrite looks like. I think this is fine to keep in so someone else who modifies CI doesn't have to look for this option if this check fails.